### PR TITLE
feat: add graphql hapify templates and helpers

### PR DESCRIPTION
**Pull request by floross**
- fix: add graphqQL templates models and resolvers
- feat: add query and resolver for nestjs graphql
- feat: add mutation to graphql templates
- feat: add documentation to hapify-templates-nestjs-graphql
- feat: write the graphql tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,29 +61,33 @@
     "@docusaurus/theme-search-algolia": "^2.0.0-beta.15",
     "@elastic/ecs-winston-format": "^1.3.1",
     "@mdx-js/react": "^1.6.21",
-    "@nestjs/axios": "^0.0.5",
-    "@nestjs/common": "^8.3.1",
-    "@nestjs/config": "^1.2.0",
-    "@nestjs/core": "^8.3.1",
-    "@nestjs/graphql": "^10.0.4",
+    "@nestjs/apollo": "^10.0.11",
```
Comment (maxmousse) [inline package.json:64]: Should we add those pakages in the README or as peer-deps of the nestjs-graphql packages or the templates pakages ?

File: workspace.json [hunk 2]
```diff
@@ -43,9 +45,43 @@
     "nestjs-core": "libs/nestjs/core",
     "nestjs-database": "libs/nestjs/database",
     "nestjs-file-storage": "libs/nestjs/file-storage",
+    "nestjs-graphql": {
```
Comment (maxmousse) [inline workspace.json]: This config should be located in its own project.json

**Review by maxmousse — APPROVED** [review]

File: workspace.json (hunk 2 above)
Comment (EdouardDem) [reply workspace.json]: I agree

**Review by EdouardDem — CHANGES_REQUESTED** [review]

**Review by EdouardDem — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (floross) [reply package.json:64]: I added it to the README

**Pull request merged**